bottlecap/src/proc/clock.rs

```diff
@@ -0,0 +1,21 @@
+use libc;
+use std::io;
```

Comment:
What's the justification on this crate? Why `libc` clock and not the standard clock?